File: Lista3/Exec1/src/exec1/Pessoa.java
/**
 * Classe que cria o básico de uma pessoa, comum a Juridica e a Fisica
 */
public class Pessoa{

    String nome, end, email;
    double tel;
    /**
     * Construtor do básico de uma pessoa (a ser expandido para pessoa fisica ou juridica por herança)
     * @param nome
     * @param endereco
     * @param email
     * @param telefone
     */
    public Pessoa(String nome, String endereco, String email, double telefone){
        this.nome = nome;
        this.end = endereco;
        this.email = email;
        this.tel = telefone;
    }

    // GETTERS
    public String getNome(){
        return this.nome;
    }
    public String getEnd(){
        return this.end;
    }
    public String getEmail(){
        return this.email;
    }
    public double getTel(){
        return this.tel;
    }

    // SETTERS
    public void setNome(String nome){
        this.nome = nome;
    } 
    public void setEnd(String endereco){
        this.end = endereco;
    }
    public void setEmail(String email){
        this.email = email;
    }
    public void setTel(double telefone){
        this.tel = telefone;
    }

}

File: Lista4/exec1/pessoa.py
class pessoa():

    def __init__(self, nome: str, endereco: str, email: str, tel: float):
        self.nome = nome
        self.end = endereco
        self.email = email
        self.tel = telefone


    # getters
    def getNome(self):
        return self.nome
    def getEnd(self):
        return self.end
    def getEmail(self):
        return self.email
    def getTel(self):
        return self.tel

    # setters
    def setNome(self, nome_):
        self.nome = nome_
    def setEnd(self, end_):
        self.end = end_
    def setEmail(self, email_):
        self.email = email_
    def setTel(self, tel_):
        self.tel = tel_